### ISSUE TITLE
QC all species already merged into stdpopsim

### ISSUE THREAD

**Issue by izabelcavassim** [popsim-consortium/stdpopsim#1069]
Here is the list of species that need to be QC'eed. 
Feel free to take one, and once you are finished let me know so I can check the box

- [x] (Multiple CONTIGS) QC for MyoLuc Species QC #899 opened on 9 Apr by lauterbur  
- [x] QC for mallard (AnaPla) Species QC #888 opened on 9 Apr by petrelharp
- [x] QC for Chlamydomonas reinhardtii Species QC #890
- [x] (@vitorpavinato) Species QC for StrAga Species QC #881 opened on 9 Apr by jeromekelleher
- [x] (@noscode) QC for Heliconius melpomene Species QC #880 opened on 9 Apr by percyfal
- [ ] (@izabelcavassim) QC for Drosophila sechellia Species QC #873 opened on 8 Apr by jradrion
- [ ] (@andrewkern) QC for AnoGam Species QC #859 opened on 8 Apr by andrewkern
- [ ] (LIFTOVER) QC for PonAbe Species QC #544 opened on 26 May 2020 by grahamgower
- [ ] (LIFTOVER) QC for HomSap Species QC #543 opened on 26 May 2020 by grahamgower
- [ ] (LIFTOVER) QC for AraTha Species QC #539 opened on 26 May 2020 by grahamgower
- [x] (@igronau) QC for stickleback Species QC #1091 opened 19 days ago by vitorpavinato

Thanks

**Comment by izabelcavassim** [comment]
@vitorpavinato can you do QC for QC for PonAbe #544  instead?

**Comment by izabelcavassim** [comment]
I am doing right now Aratha and Drosophila sechellia. Just so you know @igronau!

**Comment by igronau** [comment]
Great, thanks. I have my eye on Elise's bat species MyoLuc, but I'm not sure I'll have time this week (@lauterbur)

**Comment by grahamgower** [comment]
Hey @izabelcavassim. Thanks for keeping an eye on all of this! From memory, AraTha and HomSap were waiting on some recombination map liftovers so that we match the maps with the genome coordinates used in the species definition. We really need a vollunteer to work through what is remaining to be done with the liftovers now that Jeff has moved on.

**Comment by vitorpavinato** [comment]
Hi @izabelcavassim I can take care of __PonAbe__.

**Comment by grahamgower** [comment]
Cool, thanks @vitorpavinato. Please also see #595.

**Comment by izabelcavassim** [comment]
> Great, thanks. I have my eye on Elise's bat species MyoLuc, but I'm not sure I'll have time this week (@lauterbur)

We just need to figure out how to simplify the multiple contigs somehow, if we can't then we just don't add it to the catalog now.

**Comment by igronau** [comment]
I'll have time to look at it next week. I saw that there many contigs, but maybe we can keep it this way. If there is enough additional information, such as mutation rate and a demographic model, then we should be able to allow adding species without fully assembled genomes.

**Comment by petrelharp** [comment]
I think there's no demographic model. It would be nice to have, but there
were also concerns about adding to the load time of the library?

**Comment by vitorpavinato** [comment]
Can we only include some of the contigs/scaffolds? The largest ones (> 10Mb)?

**Comment by petrelharp** [comment]
That's a good idea? A missing piece here is: what's the use case? Like, what do we want to do with this?

**Comment by vitorpavinato** [comment]
I wasn't sure if a lot of o scaffolds would cause a slow species loading. So I tried to include the species I am working on ( [_Belgica antarctica_](https://www.nature.com/articles/ncomms5611) for now, it only has a scaffold-level genome of 4997 scaffolds, and the recombination rate I am working on it). The CLI loaded it as it would load other species (no lag). So, I think it is better to include all contigs, instead of some. It is easy to sum all the contig lengths and parse the length as the chromosome size if someone wants to simulate the whole genome.

**Comment by izabelcavassim** [comment]
I think the problem here @vitorpavinato is if we start adding a bunch of draft assemblies and then it would hurt the storage and loading. But maybe we can talk about that tomorrow in our biweekly meeting?
